DSCResources/MSFT_xDnsServerForwarder/MSFT_xDnsServerForwarder.schema.mof

```diff
@@ -0,0 +1,7 @@
+[ClassVersion("1.0.0"), FriendlyName("xDnsServerForwarder")]
+class MSFT_xDnsServerForwarder :  OMI_BaseResource
+{
+    [Key] String IPAddress;
+    [Write] Boolean RemoveAll;
```

Comment:
@PlagueHO 
Combining the single instance pattern and the membership pattern used in other resources would give us a schema like this.

This would allow the resource to be idempotent (specifically, prevent the configuration from churning on rerun), and allow the flexibility that @corydwood wants.  As well as using patterns DSC users should be familiar with.
```
xDnsServerForwarder RemoveAllForwardersfromDNSServer1 {
    [Key, Description("Specifies the resource is a single instance, the value must be 'Yes'"), ValueMap{"Yes"}, Values{"Yes"}] String IsSingleInstance;
    [Write, Description("IPAddresses of Forwarder in DNS Server which should match  exactly")] String IPAddress[];
    [Write, Description("Forwarder in DNS Server should include these IPAddresses")] String IPAddressToInclude[];
    [Write, Description("Forwarder in DNS Server should include these IPAddresses")] String IPAddressToExclude[];
}
```